code/__DEFINES/genetics.dm

```diff
@@ -81,6 +81,9 @@
 #define ORGAN_ORGANIC   1
 #define ORGAN_ROBOTIC   2
 
+#define ORGAN_FINE			1 //Attatched
+#define ORGAN_AUGMENTABLE	2 //Attatched + Augmentable 
```

Comment:
.. What?
Augment flag is to check if limb muscles are severed, effectively making the limb unusable.